index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+  <style>
+      *{
+    margin-left: 30px;
+    padding: 0;
+}
+
+body{
+    background-color:#eee ;
+
+}
+ul{
+  text-decoration: none;
+}
+
+#nav-bar{
+    top: 0px;
+    display: flex;
+    width: 100%;
+}
+
+@media print {
+    body {
+        font-size: 12pt;
+    }
+}
+
+
+@media only screen and (max-width: 80px){
+    body{
+        background-color: gray;
+    }
+
+}
+
+
+@media only screen and (max-width: 800px){
+    #nav-bar{
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+  .product{
+    border: solid #f000f0 3px;
+    border-radius: 2px;
+    text-align: center;
+  }
+    
+}
+    </style>
+  </head>
+
+  <body>
+
+    <header id="header">
+
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png">
+
+      <nav id="nav-bar">
+        <ul>
+          <a class="nav-link" href="#features">Features</a>
+          <a class="nav-link" href="#howitworks">How It works</a>
+          <a class="nav-link" href="#pricing">Pricing</a>
+        </ul>
+      </nav>
+
+    </header>
+
+    <div>
+      <h1>Handcrafted, home-made masterpieces</h1>
+
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="GET">
+        <input type="email" name="email" id="email" placeholder="Enter your email address">
+        <button type="submit" id="submit">Submit</button>
+      </form>
+
+    </div>
```

Comment:
I think it would be better to use semantic tags here. instead of div, you can use section tag.